sim/paladin/shield_of_righteousness.go

```diff
@@ -0,0 +1,49 @@
+package paladin
+
+import (
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+	"time"
+)
+
+func (paladin *Paladin) registerShieldOfRighteousness() {
+	if !paladin.hasRune(proto.PaladinRune_RuneCloakShieldOfRighteousness) {
+		return
+	}
+
+	level := float64(paladin.Level)
+
+	// Base damage formula from wowhead tooltip:
+	// https://www.wowhead.com/classic/spell=440658/shield-of-righteousness
+	damage := 179.0 * (9.046514 + 0.676562*level + 0.019349*level*level) / 100.0
```

Comment:
Every class has a function `baseRuneAbilityDamage()` that you can use to get this

Reply:
Added baseRuneAbilityDamage and used it here.